sale_order_digitized_signature/__init__.py

```diff
@@ -1 +1,4 @@
+# Copyright 2017 Tecnativa - Vicent Cubells
```

Comment:
2017 --> 2019

Reply:
No, this copyright mustn't be changed if:

* The file is not modified
* The copyright holder doesn't touch the file (which is the case).

In fact, `__init__` files shouldn't contain copyright.

In the rest of the files, it's legit to add the new copyright if the contributor touches the file.